infrastructure/front-door-docs.tf

```diff
@@ -0,0 +1,131 @@
+resource "azurerm_cdn_frontdoor_profile" "cdn_frontdoor_profile" {
+  name                = var.fd_name
+  resource_group_name = data.azurerm_resource_group.frontdoor.name
+  sku_name            = "Standard_AzureFrontDoor"
+
+  tags = local.tags
+}
+
+resource "azurerm_cdn_frontdoor_endpoint" "cdn_frontdoor_endpoint" {
+  name                     = var.fd_name
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.cdn_frontdoor_profile.id
+
+  tags = local.tags
+}
+
+resource "azurerm_cdn_frontdoor_origin_group" "cdn_frontdoor_origin_group" {
+  name                     = var.fd_name
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.cdn_frontdoor_profile.id
+  session_affinity_enabled = true
+
+  # restore_traffic_time_to_healed_or_new_endpoint_in_minutes = 10
+
+  health_probe {
+    interval_in_seconds = 240
+    path                = "/healthProbe"
```

Comment:
The existing one seems to just have `/` for path, will `healthProbe` work for storage?